src/odbc.h

```diff
@@ -168,7 +168,7 @@ typedef struct StatementData {
   SQLTCHAR *column    = NULL;
   SQLTCHAR *procedure = NULL;
 
-  SQLRETURN sqlReturnCode;
+  // SQLRETURN sqlReturnCode;
```

Comment:
Why not remove?

Reply:
I was unsure whether I added it on the object for a particular reason, so I commented out in case I needed to put it back in. Then forgot to delete it when it looked like I could go with all local variables.